homeassistant/components/ambient_station/sensor.py

```diff
@@ -241,36 +255,42 @@
         name="Max Gust",
         icon="mdi:weather-windy",
         native_unit_of_measurement=SPEED_MILES_PER_HOUR,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
```

Comment:
This is not total increasing. Wind speed is a measurement

Reply:
Are you sure? This number will continue to go up – every time a new maximum wind gust is detected, that value will be collected here.￼

Reply:
So is this Max gust ever ever? Not like Max gust this week, this month, this year?

Reply:
@Adminiuga It's per day.

Reply:
IMO it should not be TOTAL_INCREASING. Total increasing is used for things like your total energy meter. You consume energy, total consumption is ever increasing

Reply:
Ah, okay – didn't realize it was increasing forever. Just `STATE_CLASS_MEASUREMENT`, then?

Reply:
Total increasing is for metered entities.